deploy/kubernetes/Readme.md

```diff
@@ -0,0 +1,35 @@
+# Deploy Tracee on Kubernetes
+
+> NOTE `Minimal Requirements to run tracee in the node`
+````
+https://aquasecurity.github.io/tracee/install/prerequisites/
+````
+
+1. Create a ConfigMap that will hold the templates files for tracee-rules
```

Comment:
ideally, after my suggestions below, this entire example could be boiled down to `kubectl apply -f ./deploy`

Reply:
in the future we could have deploy/helm, similar folder starboard struct. Do you want to remove the kubernetes folder ?